<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecipeViewer</class>
 <widget class="QWidget" name="RecipeViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>440</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Просмотр блюда</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/ico/ico/recipe book.png</normaloff>:/ico/ico/recipe book.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="photoLabel">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>125</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>125</height>
        </size>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="src.qrc">:/ico/ico/camera.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="recipeNameLabel">
         <property name="text">
          <string>Название рецепта</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="recipeDescriptionLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Описание рецепта</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="recipeEditButton">
       <property name="text">
        <string>Редактировать рецепт</string>
       </property>
       <property name="icon">
        <iconset resource="src.qrc">
         <normaloff>:/ico/ico/edit.png</normaloff>:/ico/ico/edit.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="recipeDeleteButton">
       <property name="text">
        <string>Удалить рецепт</string>
       </property>
       <property name="icon">
        <iconset resource="src.qrc">
         <normaloff>:/ico/ico/trash.png</normaloff>:/ico/ico/trash.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ингредиенты</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="ingredientsList"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="toStepsViewerButton">
     <property name="text">
      <string>Перейти к приготовлению</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
